public/js/logic.js

```diff
@@ -15,42 +15,58 @@ $(document).ready(function(){
 
   // Add a new chest
   $("#add").click(function(){
-    container.append(chest);
-    chestCount++;
-    localStorage.setItem('count', chestCount);
+    addChests(1);
   });
 
   // Remove last chest
   $("#remove").click(function(){
-    if (chestCount > 0) {
-      container.children().last().remove();
-      chestCount--;
-      localStorage.setItem('count', chestCount);
-    }
+    deleteChests(1);
```

Comment:
Could you make this function "removeChests" so that the language matches the payload options and the UI?